tsconfig.base.json

```diff
@@ -27,7 +27,8 @@
 			"@infinum/ngx-nuts-and-bolts/testing-utils": ["libs/ngx-nuts-and-bolts/testing-utils/src/index.ts"],
 			"@ngx-nuts-and-bolts/environment-variables-example-app-base": [
 				"libs/environment-variables-example-app-base/src/index.ts"
-			]
+			],
+			"schematics": ["libs/schematics/src/index.ts"]
```

Comment:
We could maybe have a nicer name for this, e.g.

`@ngx-nuts-and-bolts/schematics`